src/Core/Localization/CLDR/Locale.php

```diff
@@ -256,7 +256,7 @@ public function getCurrencyPattern($numberingSystem = null)
             throw new LocalizationException('No currency pattern found for numbering system: ' . $numberingSystem);
         }
 
-        return $this->currencyPatterns[$numberingSystem];
+        return $this->currencyPatterns[$numberingSystem] ?? '';
```

Comment:
```suggestion
        return $this->currencyPatterns[$numberingSystem];
```
The previous isset make the job